client/src/helpers/api.ts

```diff
@@ -230,14 +241,50 @@ const updateServiceRequestStatus = async (user, serviceRequest, status, note) =>
     .catch(errorCallback);
 };
 
+const addPatientToGroup = async (groupId: string, parentId: string) => {
+  const groupResult = await getGroupById(groupId);
+  const group: Group = get(groupResult, 'payload.data.entry[0].resource', null);
+  if (!group) {
+    return Promise.reject(new Error('groupId is invalid'));
+  }
+
+  group.member.push({
+    entity: {
+      reference: `Patient/${parentId}`,
+    },
+  });
+
+  return Http.secureClinAxios.put(`${window.CLIN.fhirBaseUrl}/Group/${groupId}`, group)
+    .then(successCallback).catch(errorCallback);
```

Comment:
Here, you should also: 
* Read the parent's `Patient` resource
* Read the parent's family group (you can use `Patient.family-id`)
* For each member (Patient) of that group(the parent included), update their `family-id` to reference this group.
* Finally, you should delete the parent's group, because it's not needed anymore.

Let me know on slack if you need more info